user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,68 @@
-# Welcome to the Strapi user documentation!
+# Welcome to the Strapi user guide!
 
-**Strapi is the open-source [Headless CMS](https://strapi.io) developers love.**
\ No newline at end of file
+This user guide contains the functional documentation related to all features available in the admin panel of your Strapi application. 
+
+![Homepage of the Admin Panel](../assets/getting-started/admin-panel-homepage.png)
+
+Before going any further into this user guide, we recommend you to acknowledge the main concepts below. They will help you to understand how Strapi works, and ensure a smooth Strapi experience.
+
+- **Development environment or Production** <br> When starting working on your application, it is in development environment, which is the status for the structuration and configuration phrase of your application and its content. After deploying your application, it is in production. This status change impacts how you can use your Strapi application, as some features are only available in development environment, such as the Content-Types Builder. In the user guide, the availability or not of a feature, depending on the application status, is always mentioned in the feature's introduction.
```

Comment:
```diff
- and configuration phrase of your application
+ and configuration phase of your application
```

Personally, I would have used `step` instead of `phase`, I think it's more common.
But not sure at all it's better.

Reply:
```diff
- application and its content
+ application and its content [...contribution?]
```

I feel like the sentence is not finished, what do you think?

Reply:
```diff
- for the structuration and configuration phrase of your application and its content.
+ for the content structuration and application configuration phrase.
```

Reply:
> ```diff
> - **Development environment or Production**
> + **Development environment or Production environment**
> OR
> + **Development or Production environment**
> ```
> 
> Just **Development environment or Production** sounds strange from my point of view, cause they are both environements.

I would suggest a `Development, Staging, or Production environment` and adding in the staging to the line: `After deploying your application, it is in production or staging`

Reply:
> ```diff
> - and configuration phrase of your application
> + and configuration phase of your application
> ```
> 
> Personally, I would have used `step` instead of `phase`, I think it's more common.
> But not sure at all it's better.

I'm fine with phase here but there was a small typo.